app/src/highlighter/index.ts

```diff
@@ -240,6 +240,19 @@ const extensionModes: ReadonlyArray<IModeDefinition> = [
       '.vb': 'text/x-vb',
     },
   },
+  {
+    install: () => import('codemirror/mode/fortran/fortran'),
+    mappings: {
+      '.f': 'text/x-fortran',
+      '.f90': 'text/x-fortran',
```

Comment:
Please add a `fortran.f90` In your Highlighter tests PR. Follows our convention for other multi-extension languages

Reply:
@say25 I have added the file for you.